lib/alchemy/mount_point.rb

```diff
@@ -23,22 +21,10 @@ def get(remove_leading_slash_if_blank = true)
       # Returns the mount point path from the Rails app routes.
       #
       def path
-        match = File.read(routes_file_path).match(MOUNT_POINT_REGEXP)
-        if match.nil?
-          raise NotMountedError
-        else
-          match[1]
-        end
-      end
-
-      private
-
-      def routes_file_path
-        if Rails.root
-          Rails.root.join('config/routes.rb')
-        else
-          'config/routes.rb'
-        end
+        all_routes = Rails.application.routes.routes
+        alchemy_route = all_routes.find { |r| r.app.app == Alchemy::Engine }
```

Comment:
Let's use the actual `Alchemy::Engine.engine_name` here instead of class comparision